frontend/src/app/util/wasm.cljs

```diff
@@ -0,0 +1,23 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) KALEIDOS INC
+
+(ns app.util.wasm
+  (:require
+   ["./renderer/renderer" :as renderer]
+   ["./renderer/renderer.js$default" :as renderer-init]
+   [beicon.v2.core :as rx]
+   [potok.v2.core :as ptk]))
+
+(defn foo [] (renderer/print "Lorem ipsum"))
+
+(defn init
+  []
+  (ptk/reify ::init
+    ptk/WatchEvent
+    (watch [_ _ _] ;; TODO: mirar la docu de potok
+      (->> (rx/from (renderer-init))
+           (rx/tap foo)
```

Comment:
I know that this code is still an experiment, and it is ok to have this on app.util.wasm but I think on the end, it should not be under `app.util`. We probably need to organize it under `app` (I mean something like `app.renderer-v2`.
There is NO need to make changes now, but we do have to take this into account for the future to organize the code well.